pkgs/development/libraries/swift-corelibs-libdispatch/default.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs, lib, clangStdenv, fetchFromGitHub
+, cmake, ninja
+, overrideCC
+, clang ? pkgs.clang_10
+, libbsd
+, libsystemtap}:
+
+let
+  version = "5.5";
+in clangStdenv.mkDerivation {
+  pname = "swift-corelibs-libdispatch";
+  inherit version;
+
+  outputs = [ "dev" "out" "man" ];
```

Comment:
`out` should be listed first, otherwise `nix-build -A swift-corelibs-libdispatch` will symlink the dev output into the PWD.

Reply:
TIL!